### ISSUE TITLE
Login Error

### ISSUE THREAD

**Issue by dinamitebla** [gicamm/homeassistant-comelit#65]
I've followed the steps in the wiki but when I restart home assistant I get a bunch of warnings and two errors password related errors (I think). [home-assistant.log](https://github.com/gicamm/homeassistant-comelit/files/15052790/home-assistant.log)

My configuration.yaml file content is:
comelit:
  hub:
    host: 192.168.2.228
    port: 1883
    mqtt-user: hsrv-user
    mqtt-password: sf1nE9bjPc
    username: ******** [the email that I use to login in the app]
    password: ******** [the password that I use to login in the app]
    serial: *********
    client: homeassistant
    
My password ends with a "{", could it be the cause of the issue? Thanks in advance, best regards.

**Comment by dinamitebla** [comment]
My bad, the username and the password were not the same ones that you use in the app. After fixing the credentials I've encountered another error (I'm sorry but I didn't save it) but I've fixed it by changing `return 100 - self._position` at line 45 in the cover.py file to `return self._position` because in my case self._position is null. Should I open a different issue to report the bug and close this one?
